### PR TITLE
switch gitter to discord for drone.

### PR DESCRIPTION
**Pull request by appleboy**
cc @bkcsoft @lunny @tboerger

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by pgaskin** [comment]
LGTM

**Comment by appleboy** [comment]
make L-GTM work

File: .editorconfig [hunk 1]
```diff
@@ -15,10 +15,14 @@ indent_size = 8
 indent_style = tab
 indent_size = 4
 
-[*.{less,yml}]
```
Comment (tboerger) [inline .editorconfig:18]: We are using the same editorconfig within all repos, maybe that should be fixed on the other repos as well.

Comment (appleboy) [reply .editorconfig:18]: @tboerger Yes, I already update all repos.

**Pull request merged**